setup.py

```diff
@@ -24,14 +25,15 @@
     author=pkgmeta['__author__'],
     author_email='contact@saimon.org',
     description='Simple static gallery generator',
-    long_description=README + '\n' + CHANGELOG,
+    long_description=README,
     packages=find_packages(exclude=['tests*']),
     zip_safe=False,
     include_package_data=True,
     platforms='any',
+    python_requires='>=3.5',
```

Comment:
Good idea to include this!

It might be an idea to release a version for 2.7 and 3.4+ first (eg. 
`python_requires='>=2.7.10, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',`), see "Dropping a Python release" in https://hackernoon.com/phasing-out-python-runtimes-gracefully-956f112f33c4.

Reply:
Ok, thanks for the idea!